### ISSUE TITLE
Not able to show list inside Scrollview and flatlist

### ISSUE THREAD

**Issue by sanju280696** [PedroBern/react-native-collapsible-tab-view#298]
Hello,
I'm facing issue with Tabs.flatlist or Tabs.Scrollview. in some android device not able to see list inside Tabs. Here i have attached screenshot please check.

![WhatsApp Image 2022-08-30 at 11 34 26 AM (1)](https://user-images.githubusercontent.com/49225170/187362133-f09e3a84-cc94-42a0-8729-b13c8d8da80e.jpeg)

Let me know if anyone have idea.

Thanks

**Comment by Mehdi-KHALFALLAH** [comment]
I had the same problem, upgraded to react-native 0.72 and got this issue. I got invisible flatlist with elements which onPress they navigate to corresponding screen, but when I click inside the flatlist , it really presses on an element and navigate to the its screen.

**Comment by Tobi-Julius** [comment]
I think I've gotten a solution to the error, at least it work perfectly for me now, you've to set heightHeader to a specific value it's a must if you want to avoid seeing blank but clickable flatlist/scroll view/sectionlist, 
The reason for this is that , if you check the node module/react native collapsible tab view/src/container..tax
In your code editor search for animatedFlatlist, it should be in 2 places, where it's imported as animatedFlatlist and where it's used, check styles where it's used, you should see "pagerzStylez", when you check the pagerzStylez, you'll notice some animation for opacity in pagerOpacity 
![16758979706354741041842869741339](https://user-images.githubusercontent.com/70627316/217672837-54d33e17-95e4-49fd-8f2f-087158e40ecd.jpg)
Not setting headHeight which is equal to the initialHeadHeader and index.value can probably be null or undefined at initial render, making pagerOpacity value to zero, applying that to the flatlist, the content will be there but it'll not be visible, 
Bottom line:- adding headHeight will make the bug go away
It worked for me, if you test it on your end and it works, remember to close this issue
Thank you

**Comment by azzk-stacked** [comment]
For me setting headerHeight to some fixed value didn't work, but, setting cancelLazyFadeIn={true} did work. I didn't verify, but I feel like  @Tobi-Julius 's saying that it's an opacity is correct. Here is a video of my issue: 

https://github.com/PedroBern/react-native-collapsible-tab-view/assets/148608888/9bfb6221-c3db-4a63-b8e9-f43a154b3b06

**Comment by gregavola** [comment]
Having this same issue only Android - anyone got a fix? Worked in RN 0.71.x, but after upgrading to latestt it won't show the contents of the flatlist (ONLY if i just put Tabs.Flatlist in side the Tab, nothing else (no flex:1 views etc)
